packages/contracts/script/DeploymentConfig.s.sol

```diff
@@ -42,11 +42,15 @@ contract DeploymentConfig is Script {
     address internal SNT_ADDRESS_MAINNET = 0x744d70FDBE2Ba4CF95131626614a1763DF805B9E;
     // solhint-disable-next-line var-name-mixedcase
     address internal SNT_ADDRESS_OPTIMISM_MAINNET = 0x650AF3C15AF43dcB218406d30784416D64Cfb6B2;
+    // solhint-disable-next-line var-name-mixedcase
+    address internal SNT_ADDRESS_OPTIMISM_GOERLI = 0xcAD273fA2bb77875333439FDf4417D995159c3E1;
```

Comment:
Non-blocking: will you be publishing and verifying the source code too https://goerli-optimism.etherscan.io/address/0xcAD273fA2bb77875333439FDf4417D995159c3E1#code.

Reply:
Need to look into this again, but it seems there's no API keys on testnet etherscans so have to check if verification is even enabled there.